schemas/fluidity_options.rnc

```diff
@@ -3562,12 +3657,98 @@ prognostic_detector_options = detector_options_enabled_default
 diagnostic_detector_options = detector_options_enabled_default
 prescribed_detector_options = detector_options_disabled_default
 
+# Options for whether a field is to be included in particles.
+particle_options_enabled_default = 
+   (
+      ## Specify what is added to particles
```

Comment:
This is a bit vague. Perhaps change to "Specify whether or not values for this field are interpolated to particle locations."